### PR TITLE
docs: issue template

### PR DESCRIPTION
**Pull request by caarlos0**
let me know if you think we can add/remove stuff from this...

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    validations:
+      required: true
+  - type: textarea
+    id: reproducible
+    attributes:
+      label: How can we reproduce this?
+      description: |
+        Please share a public repository that reproduces the issue.
+        Make sure to make the reproducible as concise as possible,
+        with only the minumum required code to reproduce the issue.
```
Comment (muesli) [inline .github/ISSUE_TEMPLATE/bug.yml:23]: Typo in "minimum".

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+    validations:
+      required: true
+  - type: textarea
+    id: reproducible
+    attributes:
+      label: How can we reproduce this?
+      description: |
+        Please share a public repository that reproduces the issue.
```
Comment (muesli) [inline .github/ISSUE_TEMPLATE/bug.yml:21]: Maybe make it sound a little less restrictive: "share a code snippet, gist, or public repo" - depending on the size of the problem a small snippet within the ticket is fine with me.

Comment (caarlos0) [reply .github/ISSUE_TEMPLATE/bug.yml:21]: ah, good one

**Review by bashbunni — APPROVED** [review]
This is great! Thanks carlos

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
+name: Bug Report
+description: File a bug report
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Please fill the form below.
```
Comment (muesli) [inline .github/ISSUE_TEMPLATE/bug.yml:8]: Should probably read "Please fill out the form below."?

**Comment by muesli** [comment]
I'm all for this. We should probably add similar templates for `lipgloss` and `bubbles`, too.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+- name: Slack
+  url: https://charm.sh/slack
```
Comment (muesli) [inline .github/ISSUE_TEMPLATE/config.yml:4]: Needs to point to chat/discord now!

Comment (caarlos0) [reply .github/ISSUE_TEMPLATE/config.yml:4]: 02b6aa2

**Pull request merged**